rustls-platform-verifier/src/tests/verification_mock/mod.rs

```diff
@@ -120,41 +120,47 @@ mock_root_test_cases! {
         reference_id: EXAMPLE_COM,
         chain: &[ROOT1_INT1_EXAMPLE_COM_GOOD, ROOT1_INT1],
         stapled_ocsp: None,
+        verification_time: verification_time(),
```

Comment:
note: it looks a little silly to have a `verification_time` field on the `TestCase` struct, but then to populate it universally with `verification_time()` - this is done so that we can use a different date in the past for the stapled OCSP revocation test case. Enabling that is a WIP but I left this groundwork in place.

Reply:
That seems reasonable to me. It seems plausible we might want to expand testing in the future to check prior-to validity dates, etc, so I think this is definitely worthwhile.